TShockAPI/Bouncer.cs

```diff
@@ -2491,6 +2491,11 @@ internal void OnPlaceItemFrame(object sender, GetDataHandlers.PlaceItemFrameEven
 
 			if (!args.Player.HasBuildPermission(args.X, args.Y))
 			{
+				int num = Item.NewItem(new EntitySource_DebugCommand(), (args.X * 16) + 8, (args.Y * 16) + 8, args.Player.TPlayer.width, args.Player.TPlayer.height, args.ItemID, args.Stack, noBroadcast: true, args.Prefix, noGrabDelay: true);
```

Comment:
```suggestion
				int num = Item.NewItem(null, (args.X * 16) + 8, (args.Y * 16) + 8, args.Player.TPlayer.width, args.Player.TPlayer.height, args.ItemID, args.Stack, noBroadcast: true, args.Prefix, noGrabDelay: true);
```
Better to leave item source null rather than say it's from something it's not

Reply:
good point which making it clean 👍